### ISSUE TITLE
Decrediton not accounting ticket value correctly

### ISSUE THREAD

**Issue by psmithson** [decred/decrediton#1637]
Hi,

Congrats on all your work on DCR! 

I bought 2 tickets today, but Decrediton 1.2.1 only recognizes one of the 2 tickets in my balance. I'm sending some images.

See how the "Current Total Balance" and also in my "Locked" balance does not match the number of tickets in "Recent Staking Activity".

![problem](https://user-images.githubusercontent.com/1484015/45120820-dd079200-b124-11e8-8941-94e3bcc3b77a.JPG)

The "imported" account is also failing to account for 1 of the tickets.

![imported](https://user-images.githubusercontent.com/1484015/45120878-0de7c700-b125-11e8-93a3-b0c92fbfd0a7.JPG)

Fortunately my ticket view correctly shows 2 tickets.

![tickets](https://user-images.githubusercontent.com/1484015/45120949-42f41980-b125-11e8-91a6-06ead6013a03.JPG)

Thanks, hope you can help.

**Comment by matheusd** [comment]
Thanks for the report, I'll try to reproduce, but see if the balances correct after both tickets are mined.

**Comment by psmithson** [comment]
I reloaded Decrediton, and both tickets are now gone and my balance was updated correctly with the amount they help. It seems I might have bought the tickets exactly when the ticket price changed, so Decrediton allowed the tickets to "go through" but the blockchain blocked the transaction because it was not enough to cover the ticket price. I got another ticket now and it worked fine.

So yeah, it seems that when you execute very close to the price change this can happen.
